src/PoshNotify.psm1

```diff
@@ -2,10 +2,18 @@ if($IsMacOS -and !(Get-Module MacNotify -list)) {
     Install-Module MacNotify
 }
 
+if($IsMacOS -and (Get-Module MacNotify -list)) {
+    Update-Module MacNotify
```

Comment:
Could we maybe use `Find-Module` to check if there's a newer version before executing the `Update-Module`?

Reply:
Hmmm, that's a really good idea.

Reply:
also... I'm not sure of a good xplat way to do this... but testing for an internet connection before the `Find-Module` would be ideal.